src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Shape {
+    String color;
+    double area;
```

Comment:
what about access modifiers should they be default ? 
what happens if you move your child classes to a different package ?